presto-parser/src/main/java/com/facebook/presto/sql/tree/RollbackToSavepoint.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public final class RollbackToSavepoint
```

Comment:
This seems like a rather arbitrary definition. If this syntax was `RESTORE TO SAVEPOINT` instead of `ROLLBACK TO SAVEPOINT` would you still say should be the same AST node?

For example, should we use the same node for `DROP TABLE` and `DROP SCHEMA`?